handler/src/main/java/io/netty/handler/ssl/SslHandler.java

```diff
@@ -429,6 +429,10 @@ public void handlerRemoved0(ChannelHandlerContext ctx) throws Exception {
             // Check if queue is not empty first because create a new ChannelException is expensive
             pendingUnencryptedWrites.removeAndFailAll(new ChannelException("Pending write on removal of SslHandler"));
         }
+        if (engine instanceof OpenSslEngine) {
```

Comment:
@normanmaurer Yeah I noticed it after .... the commit message is fine as-is.  SGTM, thanks!